spec/system/jobseekers/jobseekers_can_manage_a_profile_spec.rb

```diff
@@ -127,19 +127,29 @@
       end
     end
 
-    describe "personal details if the jobseeker has a blank previous job application" do
-      let!(:previous_application) { create(:job_application, :status_draft, jobseeker:, first_name: nil, last_name: nil, phone_number: "01234567890") }
+    describe "personal details if the jobseeker has a previously submitted job application" do
+      let!(:previous_application) { create(:job_application, :status_submitted, jobseeker:, first_name: "Alfred", last_name: "Accelsior", phone_number: "01234567890") }
 
       before do
         visit jobseekers_profile_path
       end
 
       it "prefills the form with the jobseeker's provided personal details" do
         expect(page).to have_content(previous_application.phone_number)
+        expect(page).to have_content(previous_application.first_name)
+        expect(page).to have_content(previous_application.last_name)
```

Comment:
I'm not a fan of this pattern - I worry that the values might  be nil or blank (either now or in the future) at which point the assertion isn't useful. I would much prefer to assert actual values (or let variables)

Reply:
Didn't think about it, but now that you mention it, it is a good point.

Reply:
Good point, have fixed this one!